system/Database/Query.php

```diff
@@ -324,10 +318,11 @@ protected function matchNamedBinds(string $sql, array $binds): string
                 $escapedValue = '(' . implode(',', $escapedValue) . ')';
             }
 
-            $replacers[":{$placeholder}:"] = $escapedValue;
+            $replacersFrom[] = ":{$placeholder}:";
+            $replacersTo[]   = $escapedValue;
         }
 
-        return strtr($sql, $replacers);
+        return str_replace($replacersFrom, $replacersTo, $sql);
```

Comment:
Why do you replace `strtr()` with `str_replace()`?

Reply:
Per my commit diff:

> strtr() is purposed for single-character replacements, and less performant when used otherwise.